libs/domain/site/notification-center/src/notification-center.component.ts

```diff
@@ -1,29 +1,26 @@
 import { resolve } from '@spryker-oryx/di';
 import { ContentMixin, defaultOptions } from '@spryker-oryx/experience';
-import { NotificationService as SiteNotificationService } from '@spryker-oryx/site';
+import { NotificationService } from '@spryker-oryx/site';
 import {
   NotificationCenterComponent,
   NotificationPosition,
 } from '@spryker-oryx/ui/notification-center';
 import { hydratable, subscribe } from '@spryker-oryx/utilities';
 import { html, LitElement, TemplateResult } from 'lit';
-import { ifDefined } from 'lit/directives/if-defined.js';
 import { createRef, ref } from 'lit/directives/ref.js';
 import { tap } from 'rxjs';
 import { SiteNotificationCenterOptions } from './notification-center.model';
 
 @defaultOptions({
-  marginBlock: '40px',
-  marginInline: '30px',
-  maxWidth: '470px',
-  enableStacking: true,
   position: NotificationPosition.BottomStart,
+  enableStacking: true,
+  defaultAutoCloseTime: 8,
```

Comment:
I believe that milliseconds is a standard

Reply:
And seconds are easier for business users.